File: src/main/java/com/randomperson22/piggymodtest/entities/reindessa/EntityReindessa.java
package com.randomperson22.piggymodtest.entities.reindessa;

import com.randomperson22.piggymodtest.entities.EntityPiggyBase;
import com.randomperson22.piggymodtest.init.ModSounds;
import net.minecraft.util.SoundEvent;
import net.minecraft.world.World;

public class EntityReindessa extends EntityPiggyBase {
    public EntityReindessa(World worldIn) {
        super(worldIn);
    }

    @Override
    public SoundEvent getJumpscareSound() {
        return ModSounds.ENTITY_DESSA_JUMPSCARE;
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/entities/reindessa/EntityReindessa.kt
package com.randomperson22.piggymodtest.entities.reindessa

import com.randomperson22.piggymodtest.entities.EntityPiggyBase
import com.randomperson22.piggymodtest.init.ModSounds
import net.minecraft.util.SoundEvent
import net.minecraft.world.World

class EntityReindessa(worldIn: World) : EntityPiggyBase(worldIn) {
    override val jumpscareSound: SoundEvent
        get() = ModSounds.ENTITY_DESSA_JUMPSCARE
}